app/src/app/views/stix/collection/collection-import/collection-import-workflow/collection-import.component.ts

```diff
@@ -249,6 +257,56 @@ export class CollectionImportComponent implements OnInit {
         })
     }
 
+    /**
+     * Determine if the user can force import a collection after the post collection
+     * call fails. Users may force an import when:
+     *   1. the collection bundle is already in the database (user is re-importing the bundle)
+     *   2. the collection bundle has one or more objects with an invalid ATT&CK spec version.
+     * @param error the resulting error from previewing the collection
+     * @returns true if the user can force import the collection; false otherwise
+     */
+    private canForceImport(err: any): boolean {
+        if (!err.error) return false;
+        if (err.error.bundleErrors.duplicateCollection ||
+            err.error.objectErrors.summary.invalidAttackSpecVersionCount) return true;
```

Comment:
I believe we need to have the opposite approach here. The user cannot force an import if:
- The bundle has more than one collection (?)
- The bundle has no collection
- The collection is badly formatted

Checking if these two conditions where it can be overridden is irrelevant if the other errors are present. So we need to check if the collection _cannot_ be force-imported.

Might need to verify exactly which errors can be overridden with the forceImport flag with @ElJocko however.

Reply:
I don't think the REST API supports importing a collection-less bundle at this point in time